universal-login-react/src/ui/Transfer/Recipient/TransferRecipient.tsx

```diff
@@ -5,13 +5,14 @@ import './../../styles/transferRecipientDefaults.css';
 import {getStyleForTopLevelComponent} from '../../../core/utils/getStyleForTopLevelComponent';
 
 export interface TransferRecipientProps {
+  symbol: string;
   onRecipientChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
   onSendClick: () => Promise<void>;
   transferDetails: Partial<TransferDetails>;
   transferRecipientClassName?: string;
 }
 
-export const TransferRecipient = ({onRecipientChange, onSendClick, transferRecipientClassName, transferDetails: {amount, currency, to}}: TransferRecipientProps) => {
+export const TransferRecipient = ({onRecipientChange, onSendClick, transferRecipientClassName, transferDetails: {amount, to}, symbol}: TransferRecipientProps) => {
```

Comment:
symbol is now part of TransferDetails?

Reply:
TransferDetails operate on tokenAddresses now